### PR TITLE
feat: new simple page template

### PR DESCRIPTION
**Pull request by leopuleo**
## Description

Adding a new dynamic page template - see Contentful **Simple page** content model.

The new content model has the following fields: 
- **Page name:** the page title
- **Slug:** the relative page path
- **Metadata - Title:** the SEO page title
- **Metadata - Description:** the SEO page description
- **Metadata - Robots status:** boolean value, useful to prevent draft or sensible pages indexing
- **Content:** the page content (markdown)
 
To handle the page content/structure, I create 3 new components: 

- `PageTitle`: the top section containing the page tile
- `PageContent`: the main content
- `Markdown`: the component used to style markdown

I decided to create new components for the following reasons: 

- don't break existing styles used across the website
- to be reused in case of future similar pages - e.g. blog posts

In case of a PR approve, I will work on refactoring existing components

### Examples:
- [Demo content](https://5fabe26e363891a4cf9c9bef--mondora.netlify.app/example-simple-page-template/)
- [Company Data page](https://5fabe26e363891a4cf9c9bef--mondora.netlify.app/company-data/)

## Bonus tracks:
- switch `JumboTitle` from H2 to H1 - every webpage need at least one Heading 1 
- unused dependencies cleanup.
- upgrade all remaining dependencies.
- Remove Google Fonts external stylesheet import - see `gatsby-plugin-google-fonts`.
- Remove Slick carousel external stylesheet import - see` gatsby-browser.js`.

## Note: 
Many file changes are due to `prettier` upgrade. 🤦

### CODE REVIEW

**Comment by manuelserradev** [comment]
The update to `prettier@2` introduced some "breaking" changes, notably the ending comma and arrow functions parenthesis. This introduce many diffs and noise.

What I suggest is to rollback to `prettier@1` or configure `prettier@2` with the same rules as for version 1, see [here](https://prettier.io/blog/2020/03/21/2.0.0.html#breaking-changes).

A further PR with new rules only should be then opened and after accepted.

File: src/components/markdown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import rehypeReact from "rehype-react";
+import styled from "styled-components";
+
+const PrimaryTitle = styled.h1`
+    font-weight: 400;
+    font-size: 2em;
```
Comment (manuelserradev) [inline src/components/markdown/index.jsx:9]: Should we use `rem`? Are we using `em` intentionally?

File: src/components/markdown/index.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+const Paragraph = styled.p`
+    font-size: calc(1em + 0.15vw);
+    line-height: 1.7em;
+    margin: 0 0 1em;
+    color: var(--text-gray);
+`;
+
+const Bold = styled.strong`
+    color: var(--text-dark-black);
```
Comment (manuelserradev) [inline src/components/markdown/index.jsx:46]: We are fragmenting colors in both css and js with theme.

My pov is that theme is a superset of css vars and should be used instead.

**Comment by leopuleo** [comment]
> The update to `prettier@2` introduced some "breaking" changes, notably the ending comma and arrow functions parenthesis. This introduce many diffs and noise.
> 
> What I suggest is to rollback to `prettier@1` or configure `prettier@2` with the same rules as for version 1, see [here](https://prettier.io/blog/2020/03/21/2.0.0.html#breaking-changes).
> 
> A further PR with new rules only should be then opened and after accepted.



> The update to `prettier@2` introduced some "breaking" changes, notably the ending comma and arrow functions parenthesis. This introduce many diffs and noise.
> 
> What I suggest is to rollback to `prettier@1` or configure `prettier@2` with the same rules as for version 1, see [here](https://prettier.io/blog/2020/03/21/2.0.0.html#breaking-changes).
> 
> A further PR with new rules only should be then opened and after accepted.

Grazie Manuel, I fixed the breaking rules. 😄

File: src/components/markdown/index.jsx (hunk 1 above)
Comment (leopuleo) [reply src/components/markdown/index.jsx:9]: I prefer `em` over `rem` for this kind of projects where you need granular control on elements size. 
Anyway, I noticed we used `pt`, `px`, `rem` and `em` on this project. 

We should adopt a strategy and stick to it, but I think this should be discussed in a separate PR.

File: src/components/markdown/index.jsx (hunk 2 above)
Comment (leopuleo) [reply src/components/markdown/index.jsx:46]: I agree. I believe it's better to discuss this change in a separate PR.

**Review by manuelserradev — APPROVED** [review]

**Pull request merged**